config/vufind/config.ini

```diff
@@ -1763,10 +1763,27 @@ hide_holdings[] = "World Wide Web"
 ;
 ; Available options:
 ;    Channels - Display links to channels of content related to record
+;    Bookplate - Display a bookplate or something similar
 ;    Similar - Similarity based on Solr lookup
 ;    WorldCatSimilar - Similarity based on WorldCat lookup
 related[] = "Similar"
 
+; URLs for bookplates if enabled in the related config above. %s represents an
+; image name or donor code that can be used to identify an image. This will
+; come from a Solr field (donor_code by default).
+; bookplate_full = https://your-institution.edu/bookplates/%s-full.jpg
+; bookplate_thumb = https://your-institution.edu/bookplates/%s-thumb.jpg
+
+; Solr field with an array of image titles, parallel with bookplate_img_names_field.
```

Comment:
It may make sense to drop the word Solr from these comments, and build this in a slightly more generic way. While I think this is most likely to be used in combination with Solr, it's conceivable that somebody might want to leverage the component to display data based on a web-scale discovery service or other data source. If you make things just slightly more generic, it should be possible to support that. Most services offer an array of fields, so if you fetch the field data using getRawData and then use the indexes configured here, the solution may be generalizable to at least some other search backends. Further flexibility might be achievable by subclassing, but we can cross that bridge when we come to it.

Reply:
I did this and I also removed the word Solr from function names.